site/src/testHelpers/storybook.tsx

```diff
@@ -45,25 +45,39 @@ export const withDashboardProvider = (
   );
 };
 
+type MessageEvent = Record<"data", string>;
+type CallbackFn = (ev?: MessageEvent) => void;
+
 export const withWebSocket = (Story: FC, { parameters }: StoryContext) => {
-  if (!parameters.webSocket) {
-    console.warn(
-      "Looks like you forgot to add websocket messages to the story",
-    );
+  const events = parameters.webSocket;
+
+  if (!events) {
+    console.warn("Your forgot to add the `parameters.webSocket` to your story");
```

Comment:
Minor typo, and `the` sounds a bit off to me:

```suggestion
    console.warn("You forgot to add `parameters.webSocket` to your story");
```